va/va_enc_vp9.h

```diff
@@ -154,8 +154,11 @@ typedef struct  _VAEncPictureParameterBufferVP9
     /** VP9 encoder may support dynamic scaling function.
      *  If enabled (enable_dynamic_scaling is set), application may request
      *  GPU encodes picture with a different resolution from the raw source.
-     *  GPU should handle the scaling process of source and
-     *  all reference frames.
+     *  Applications are expected to handle the scaling explicitly:
+     *  application should scale the raw source and send to driver what
+     *  would be encoded via frame_width_src/frame_height_src. GPU would
```

Comment:
please replace GPU to "HW" or "backend driver"

Reply:
Should I replace GPU everywhere or just here?

Reply:
@eshiryae , in current PR